data/human/human missions.txt

```diff
@@ -6891,3 +6891,370 @@ mission "Returning to Paradise"
+			choice
+				`	"I think you're right."`
+				`	"What about doing both?"`
+					goto plan
+
+			`	"And you know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
+				goto showtime
+
+			label plan
+			`	"Now you're talking! But it would take a lot of planning to do it without killing the people we are trying to liberate. You know they'll use those poor people as human shields. Say we-" A chirp from the proximity alert cuts him off.`
+
+			label showtime
+			`	"Showtime!" he exclaims, rubbing his hands together. Both of you watch as a capital ship accompanied by a small flotilla of Quicksilver escorts, all in pirate colors, lands near the security detail, probably deafening them with the sound of their repulsors. As soon as the loading ramp is down, the security detail gets to work herding the slaves from one ship to another - hitting, shoving, and shouting at anyone that moves too slowly for their liking.`
+			`	"Pigs," blurts out Bunker. He zooms in on two men talking a few meters away from everyone else. One is in a suit, the other a black hooded robe. "Must be the leaders," he whispers as if they can hear him. Just then, the hooded man looks up to the sky as if he sees something. He motions, and the Quicksilvers fire up their engines.`
+			`	"Oh, I don't like the look of this," says Bunker.`
+			choice
+				`	"Keep filming."`
+				`	"Call back the drones immediately."`
+					goto "call drones"
+				`	"I think we'll have to leave the drones behind."`
+					goto "no can do"
+				`	"Did he see us?"`
+					goto "see us"
+
+			`	"I don't know. I think we oughta' get going," says Bunker, grimacing.`
+			choice
+				`	"That's an order!"`
+				`	"Okay. Call the drones back."`
+					goto "call drones"
+
+			`	"You're the captain," Bunker responds in a sore tone. Meanwhile, you continue to watch the display. It takes a couple minutes before the last of the slaves are loaded, followed by the security team. The two men in the suit and hood are the last to board their ships.`
+			`	You barely notice the humming of thrusters from outside just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly from nowhere as they whittle the shields down in a split second. You shunt the <ship>'s repulsors to full blast to get into orbit, but another pair of Quicksilvers send another volley of particle fire that breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out underneath you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
```

Comment:
```suggestion
			`	You hear the sudden hum of thrusters from outside, just as the <ship> receives a hail of particle cannon fire from a pirate Quicksilver, appearing seemingly out of nowhere as it tears through your hull. You shunt the <ship>'s repulsors to full blast to get into orbit, but another pair of Quicksilvers sends a volley of particle fire that breaks through and takes the repulsors offline. The last thing you experience is the ship slowing to a halt, pausing in the air for just a moment before the pilot seat goes out from under you, the cockpit filled with screams as the <ship> makes a one way trip towards the surface.`
```

The repulsors were deafening in the previous text, but now you barely notice them?
I'm also suggesting that you have the Quicksilvers tear through the hull in the death scene. It would be insane to imagine particle cannons breaking through the shield of a T2 ship in a short time.

Reply:
The repulsors would be deafening... if you were close by the ships. In both cases you are far away.